bin/rails

```diff
@@ -1,8 +1,11 @@
 #!/usr/bin/env ruby
-# This command will automatically be run when you run "rails" with Rails 3 gems installed from the root of your application.
 
-ENGINE_ROOT = File.expand_path('../..', __FILE__)
-ENGINE_PATH = File.expand_path('../../lib/spree_static_content/engine', __FILE__)
+unless File.exist? 'spec/dummy/bin/rails'
+  system "bin/rake", "spec/dummy" or begin
+    warn "Automatic creation of the dummy app failed"
+    exit 1
+  end
+end
 
-require 'rails/all'
-require 'rails/engine/commands'
+Dir.chdir 'spec/dummy'
+exec 'bin/rails', *ARGV
```

Comment:
Interesting change, it may be worth porting this feature to other extensions as well